platform_versioned_docs/version-23.4.0/data/data-studios.mdx

```diff
@@ -90,7 +94,33 @@ A data studio session uses resources until it's **stopped**.
 
 Once a data studio session is in a **running** state, you can connect to it, obtain the public link to the session to share with collaborators inside your workspace, and stop it.
 
-### Connect to a data studio
+## Start an existing data studio as a new data studio
+
+This functionality is available to users with the **Maintain** role and above.
+
+You can use an existing data studio as the basis for starting a new data studio. This functionality creates a clone of the copied data studio, including its existing checkpoints, allowing you to preserve any modifications that you made to the original data studio. When you create a data studio based in this way, future changes are isolated from the original data studio.
+
+The following fields are immutable and cannot be changed:
+
+- **Data studio template**
+- **Original data studio and checkpoint**
+- **Compute environment**
+
+To create a new data studio from an existing data studio, complete the following steps:
+
+1. In a workspace, select **Data Studios**.
+1. From the list of data studios in your workspace, select the three dots next to the status message for the data studio you want to start, then select **Start as new**.
+1. Optional: Customize the following fields:
```

Comment:
```suggestion
1. This opens the **Add data studio** form. Optionally customize the following fields:
```

Reply:
@robnewman, I might rephrase this based on: https://developers.google.com/style/procedures#steps-with-results-or-justifications